### PR TITLE
Download eventLogFile feature

### PR DESCRIPTION
**Pull request by Annubis45**
Add a popup menu on eventlogfile path to let users download Event Log file with one click.

### CODE REVIEW

**Comment by tprouvot** [comment]
Hi @Annubis45,
Thanks for this PR 🙏 

Could you fill the CHANGES.md file and follow current convention for describing the new feature implementation ? 
Also it could be nice to describe it with a gif in the 'how-to' page (I can do it if you want).

Two remarks regarding the feature:
- could you add a cursor pointer to indicate the user the link is clickable ?
- Remove "Copy Id" option or rename it to "Copy Path" if you think this can be useful 

image

Thanks a lot !

**Comment by Annubis45** [comment]
> Hi @Annubis45, Thanks for this PR 🙏
> 
> Could you fill the CHANGES.md file and follow current convention for describing the new feature implementation ? Also it could be nice to describe it with a gif in the 'how-to' page (I can do it if you want).
> 
> Two remarks regarding the feature:
> 
> * could you add a cursor pointer to indicate the user the link is clickable ?
> * Remove "Copy Id" option or rename it to "Copy Path" if you think this can be useful
> 
> image
> Thanks a lot !

I've implemented your remarks and done the gif in the how to. 
I didn't know which url to put in the src of the img tag. Hope I used the right way.

**Comment by tprouvot** [comment]
@Annubis45  That's great  👏  thanks for the updates 🙏 
You're right at the beginning the images were uploaded like that but now I'm pasting the image when editing the page directly from github, it saves some storage.

One last remark, it could be great to name the downloaded file by the eventLog id, it could be easier to make the distinction when downloading multiple files, what do you think ?

**Comment by Annubis45** [comment]
@tprouvot  I've implemented your suggestion. The default name file id  : 0ATxxxxxxxxxx_LogFile.csv.

File: addon/data-load.js [hunk 1]
```diff
@@ -211,19 +211,41 @@ function renderCell(rt, cell, td) {
...
+        aDownload.textContent = "Download File";
+        aDownload.className = "download-salesforce";
+        let aDownloadIcon = document.createElement("div");
+        aDownloadIcon.className = "icon";
+        pop.appendChild(aDownload);
+        aDownload.prepend(aDownloadIcon);
+        aDownload.addEventListener("click", e => {
+          sfConn.rest(e.target.id, {responseType: "text/csv"}).then(data => {
+            let downloadLink = document.createElement("a");
+            downloadLink.download = recordId.split("EventLogFile/")[1];
```
Comment (tprouvot) [inline addon/data-load.js:228]:
```suggestion
            downloadLink.download = recordId.split("/")[6];
```

**Pull request merged**